packages/chat/components/feedback/src/feedback-container.template.ts

```diff
@@ -0,0 +1,323 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, PropertyValueMap } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { v4 as uuidv4 } from 'uuid';
+// @ts-ignore
+import styles from './feedback-container.scss?inline';
+import '@carbon/web-components/es/components/modal/index.js';
+import '@carbon/web-components/es/components/textarea/index.js';
+import '@carbon/web-components/es/components/radio-button/index.js';
+import '@carbon/web-components/es/components/button/index.js';
+import '@carbon/web-components/es/components/checkbox/index.js';
+import '@carbon/web-components/es/components/form-group/index.js';
+
+import Flag24 from '@carbon/web-components/es/icons/flag/24';
+import { FeedbackAPI } from '../../../services/feedback';
+
+/**
+ * Feedback Container component to record and give feedback on AI generated content
+ */
+export class FeedbackContainer extends LitElement {
+  static styles = styles;
+
+  /**
+   * API Key of registered application for using this component
+   */
+  @property({ attribute: 'api-key' })
+  private _api_key: string = '';
```

Comment:
You can define the type in the `@property({ type: String, ...})` instead. Is it necessary to make the